tcf_website/templates/reviews/duplicate_review_modal.html

```diff
@@ -0,0 +1,20 @@
+<!-- Delete Confirmation Modal -->
+<div class="modal fade" id="duplicate-review" tabindex="-1" role="dialog" aria-labelledby="duplicate-review-label" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Duplicate Review</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+          It looks like you've already submitted a review for this class! If you would like, you can go to your
+          original review to edit it as you see fit.
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-warning" data-dismiss="modal">Close</button>
```

Comment:
I don't think the yellow looks great considering the style guide...maybe change it to orange like the submit button?

(for any other reviewers, this is what it looks like)
![image](https://user-images.githubusercontent.com/55100084/110560074-58da5f00-8113-11eb-908e-635b1756e991.png)

Reply:
Agreed, should def fit the site theme and I made it rather quickly so I'll update it.